packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -9,8 +10,176 @@
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+## Canaries
+
+AWS Cloudwatch Synthetics are used to create *canaries*, scripts that run on a schedule and monitor websites and API endpoints. You can read more about them [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html).
```

Comment:
Amazon CloudWatch Synthetics allows you to monitor your application by continuously generating **synthetic** traffic, the traffic is generated by a **canary**, the simplest way to think of a canary is like a customer, a canary will hit your endpoint as if it was a customer allowing you to verify the health of your endpoint.